tap_tool/integrations/linkedin.py

```
@@ -57,7 +57,7 @@ def list_linkedin_accounts(access_token, company):
     )
 
     if not resp:
-        custom_error(f"error: {resp.text}", True)
+        custom_error(f"error: {resp.text}", False)
```

If you do it like this, then it does not stop execution. The runtime will continue. Maybe we should throw an Exception instead of `sys.exit`. Then it would be caught by this code 
```
except Exception as exc:
            job_failed.append(integration)
            logging.error(f"{company} {integration} job failed. The reason is {exc}")
            continue
```
 in `__init__.py`

---

I just tried, try except, but it does not catch the exception. Do @paddie  have some ideas?

---

```
if not resp:
            if resp.status_code == 401:
                  raise Exception(f"linkedin access token expired for {company}")
            else:
                   raise Exception(f"linkedin sync failed for {company}, error: {resp.text}")
```